### ISSUE TITLE
about validation loss

### ISSUE THREAD

**Issue by getstarlu145** [ThibaultGROUEIX/AtlasNet#19]
![image](https://user-images.githubusercontent.com/22369142/55785804-a5f7c580-5ae5-11e9-94c0-0cd2e685fa87.png)
I trained your code on python3.7/torch0.4.1 . It has this problem.The training loss becomes low but the validation loss increases.Do I have problem in torch vertion?

**Comment by ThibaultGROUEIX** [comment]
Hi @getstarlu145,
Yes, please update to torch v1,
Best regards,
Thibault

**Comment by getstarlu145** [comment]
Ok, it can work well now.Thank you!
